ouf.lua

```diff
@@ -274,7 +274,8 @@ local function initObject(unit, style, styleFunc, header, ...)
 
 			-- No need to enable this for *target frames.
 			if(not (unit:match('target') or suffix == 'target')) then
-				object:SetAttribute('toggleForVehicle', true)
+				-- BUG: Blizzard has changed the way vehicles work for Antoran High Command
+				object:SetAttribute('toggleForVehicle', false)
```

Comment:
Let the non-header units have vehicle support, this issue only concerns vehicles.

Reply:
This does not remove vehicle support, it changes the way it works.